peps/pep-0815.rst

```diff
@@ -514,8 +519,8 @@ The potential for improvement can be summarized as:
     work-arounds developers have been pursuing to support the rapidly
     growing AI/ML and scientific computing worlds.
 
-    — Travis Oliphant, the author of NumPy and SciPy and Chief AI
-    Architect at OpenTeams
+    — Travis Oliphant, the author of NumPy_ and SciPy_ and Chief AI
+    Architect at `OpenTeams <https://openteams.com/>`_
```

Comment:
It's a company name, I wouldn't make that a link. The first mention of NVIDIA also isn't a link, and I think that that would probably be frowned upon in a PEP.

Reply:
Right, makes sense.